main.js

```diff
@@ -184,18 +184,12 @@ CrispCache.prototype.set = function (key, value, options, callback) {
         expiresTtl = this._getDefaultExpiresTtl();
     }
 
-    if (this._lru && options.size === undefined) {
-        var errStr = 'Cache entry set without size and maxSize is enabled, key was: ' + key;
-        debug(errStr);
-        return callback(new Error(errStr));
-    }
-
     if (expiresTtl > 0) {
         var cacheEntry = new CacheEntry({
             value: value,
             staleTtl: staleTtl,
             expiresTtl: expiresTtl,
-            size: options.size
+            size: options.size || 1
```

Comment:
Can you check for undefined? Someone may want to add something to the cache and have it take up no size. Kind of a weird use case, but if they want it to not count against the quota for some reason because it's important.